modules/custom/az_cas/config/install/az_cas.settings.yml

```
@@ -1,3 +1,5 @@
 disable_login_form: true
 disable_admin_add_user_button: true
 disable_password_recovery_link: true
+guest_mode: false
+guest_auth_paths: []
```

Not sure if this is correct, but we can talk about it.
```suggestion
guest_auth_paths: {}
```